recipe/pipeline.py

```diff
@@ -30,87 +30,47 @@ def source_url(day: str) -> str:
-        "cache_location", default=f"gs://pangeo-forge-scratch/cache/{name}.zarr"
-    )
-    target_location = Parameter("target_location", default=f"gs://pangeo-forge-scratch/{name}.zarr")
-
-    @property
-    def sources(self):
-        # This can be ignored for now.
-        pass
-
-    @property
-    def targets(self):
-        # This can be ignored for now.
-        pass
-
-    def get_test_parameters(self, defaults: dict):
-        parameters = dict(defaults)  # copy the defaults
-        parameters["days"] = defaults["days"][:5]
-        parameters["cache_location"] = "memory://cache/"
-        parameters["target_location"] = "memory://target.zarr"
-        return parameters
-
-    # The `Flow` definition is where you assemble your pipeline. We recommend using
-    # Prefects Functional API: https://docs.prefect.io/core/concepts/flows.html#functional-api
-    # Everything should happen in a `with Flow(...) as flow` block, and a `flow` should be returned.
-    @property
-    def flow(self):
-        with Flow(self.name) as flow:
-            # Use map the `source_url` task to each day. This returns a mapped output,
-            # a list of string URLS. See
-            # https://docs.prefect.io/core/concepts/mapping.html#prefect-approach
-            # for more. We'll have one output URL per day.
-            sources = source_url.map(self.days)
-
-            # Map the `download` task (provided by prefect) to download the raw data
-            # into a cache.
-            # Mapped outputs (sources) can be fed straight into another Task.map call.
-            # If an input is just a regular argument that's not a mapping, it must
-            # be wrapepd in `prefect.unmapped`.
-            # https://docs.prefect.io/core/concepts/mapping.html#unmapped-inputs
-            # nc_sources will be a list of cached URLs, one per input day.
-            nc_sources = download.map(sources, cache_location=unmapped(self.cache_location))
-
-            # The individual files would be a bit too small for analysis. We'll use
-            # pangeo_forge.utils.chunk to batch them up. We can pass mapped outputs
-            # like nc_sources directly to `chunk`.
-            chunked = pangeo_forge.utils.chunk(nc_sources, size=5)
-
-            # Combine all the chunked inputs and write them to their final destination.
-            writes = combine_and_write.map(
-                chunked,
-                unmapped(self.target_location),
-                append_dim=unmapped("time"),
-                concat_dim=unmapped("time"),
-            )
-
-            # Consolidate the metadata for the final dataset.
-            consolidate_metadata(self.target_location, writes=writes)
-
-        return flow
-
-
-# pangeo-forge and Prefect require that a `flow` be present at the top-level
-# of this module.
-flow = Pipeline().flow
+days = pd.date_range("1981-09-01", "1981-09-10", freq="D").strftime("%Y-%m-%d").tolist()
+sources = list(map(source_url, days))
```

Comment:
Alongside my refactor comment about `source_url`, 

```suggestion
source_urls = [get_source_url(day) for day in days]
```

Feels more 'common' and Pythonic than the functional approach of `list(map(`